contract-creator/src/org/jetbrains/ruby/runtime/signature/server/serialisation/RTupleBuilder.java

```diff
@@ -48,15 +48,11 @@ private RTupleBuilder(ServerResponseBean bean) {
```

Comment:
Add some documentations or introduce constants and so on. This is a magic numbers here